pylabrobot/liquid_handling/backends/hamilton/STAR.py

```diff
@@ -385,10 +386,32 @@ def __init__(self, num_channels: int = 8, read_timeout: int = 5, **kwargs):
     self.dev: Optional[usb.core.Device] = None
     self._tip_types: dict[str, int] = {}
     self.num_channels = num_channels
+    self._iswap_parked: bool = False
 
     self.read_endpoint: Optional[usb.core.Endpoint] = None
     self.write_endpoint: Optional[usb.core.Endpoint] = None
 
+  @property
+  def iswap_parked(self)->bool:
```

Comment:
This is very nitpicky, and not even included in our current linting rules, but please add spaces around the arrow for consistent code style with the rest of the repo :)